expressEngine/lib/validationHandler/index.js

```diff
@@ -0,0 +1,3 @@
+import validateRequired from './validateRequired';
+
+export { validateRequired };	
```

Comment:
The name of the function is "validateReqData" in validationRequired.js and "validationRequired" in index.js.
Please export and import using the same name

Reply:
I have updated now